salt/_modules/metalk8s.py

```diff
@@ -275,3 +275,14 @@ def check_pillar_keys(keys, refresh=True, pillar=None, raise_error=True):
             return False
 
     return True
+
+
+def file_read(path):
+    """Returns the content of a given file
+
+    Arguments:
+        path (str): path to a file
+    """
+
+    with open(path, 'r') as fd:
+        return fd.read()
```

Comment:
Well, I must admit that the commit message is not really good.
But, Salt is returning something that's not parsable by the yaml filter.
It's like the whole content is escaped with quotes, the YAML filter sees only a single string, instead of a complex object and renders it badly.
Anyway, I've removed this part. :)